### PR TITLE
Initial commit creating spring boot application

### PR DESCRIPTION
**Pull request by hwilliams1**
Initial commit

### CODE REVIEW

File: routes.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+app_name: certified-copies.orders.api.ch.gov.uk
+group: api
+weight: 924
+routes:
+   1: ^/orderable/certified-copies/.*
```
Comment (nabeelamirch) [inline routes.yaml:5]: Not entirely necessary, but it says there is no new line at the end of the file

File: .gitignore [hunk 2]
```diff
@@ -21,3 +21,4 @@
 
 # virtual machine crash logs, see http://www.java.com/en/download/help/error_hotspot.xml
 hs_err_pid*
+/target/
```
Comment (nabeelamirch) [inline .gitignore:24]: Could you add .DS_Store to this and remove the .DS_Store files below

Comment (hwilliams1) [reply .gitignore:24]: Yep will do

File: routes.yaml (hunk 1 above)
Comment (hwilliams1) [reply routes.yaml:5]: A new line is forthcoming then :)

**Pull request merged**